### ISSUE TITLE
Error when running Che using CLI utility

### ISSUE THREAD

**Issue by ghost** [eclipse-che/che#2128]
Cannot run Che using the CLI on Linux.

**Reproduction Steps:**
1. Follow instruction here - https://eclipse-che.readme.io/docs/che-cli

**Expected behavior:**

Che starts

**Observed behavior:**

```
eugene@eugene:~/Stuff/che (master)$ ./che start
WARNING: No swap limit support
INFO: ECLIPSE CHE: FOUND IMAGE codenvy/che-launcher:latest
INFO: ECLIPSE CHE: LAUNCHING LAUNCHER
unknown flag: --e
See 'docker run --help'.
eugene@eugene:~/Stuff/che (master)$ 
```

The error occurs when CHE_LOCAL_CONF_DIR is exported in the system. Unsetting it fixes the problem.

**Comment by TylerJewell** [comment]
This may not be a bug.  This looks like possibly a caching issue.  The `--e` environment variable was removed from the che.sh script in a recent version.  It should only be using `--env` now when making calls to the `codenvy/che-launcher`.  And the launcher itself didn't seem to make it far enough to call the `codenvy/che-server`.  

Can you wipe your che.sh and grab the latest from master for another test.  Also, try `./che info`.

**Comment by TylerJewell** [comment]
Ok - we diagnosed this to a bug with bash "echo" command.  For some reason if `--env-file` is in a bash string, then echo interprets `-e` as a flag and ignores the `n`.  The solution is to hard code `--env-file` into the docker run syntax.

**Comment by TylerJewell** [comment]
Resolved as part of #2119

**Comment by JamesDrummond** [comment]
@TylerJewell Can you provide a status label on this?
